src/libs/SidebarUtils.js

```diff
@@ -247,7 +247,7 @@ function getOptionData(reportID) {
     if (ReportUtils.isReportMessageAttachment({text: report.lastMessageText, html: report.lastMessageHtml})) {
         lastMessageTextFromReport = `[${Localize.translateLocal('common.attachment')}]`;
     } else {
-        lastMessageTextFromReport = Str.htmlDecode(report ? report.lastMessageText : '');
+        lastMessageTextFromReport = _.get(report, 'lastMessageText', '');
```

Comment:
This line causes a crash on iOS, when we get `lastMessageText: null` then it is causing the crash

![Simulator Screen Shot - iPhone 14 - 2023-04-14 at 09 44 56](https://user-images.githubusercontent.com/35371050/231942349-c3d37ca8-bdfe-47bf-babc-3591bd08aae7.png)

cc: @tienifr @sobitneupane @pecanoro @MariaHCD

Reply:
@tienifr Let's get this sorted asap.

Reply:
Created PR https://github.com/Expensify/App/pull/17449

Reply:
@sobitneupane Thanks for your quick PR. I'm really sorry for this delay. Do you need any help?

Reply:
@tienifr If you are available, can you please review the [PR](https://github.com/Expensify/App/pull/17449)

Reply:
Yes, sure